docs/Installation-Guide.rst

```diff
@@ -1,17 +1,30 @@
 Installation Guide
 ==================
 
-This is a guide for building the LightGBM Command Line Interface (CLI). If you want to build the Python-package or R-package please refer to `Python-package`_ and `R-package`_ folders respectively.
-
 All instructions below are aimed at compiling the 64-bit version of LightGBM.
 It is worth compiling the 32-bit version only in very rare special cases involving environmental limitations.
 The 32-bit version is slow and untested, so use it at your own risk and don't forget to adjust some of the commands below when installing.
 
+By default, instructions below will use **VS Build Tools** or **make** tool to compile the code.
+It it possible to use `Ninja`_ tool instead of make on all platforms, but VS Build Tools cannot be replaced with Ninja.
```

Comment:
> ... but VS Build Tools cannot be replaced with Ninja.

Actually, [it can be](https://fekir.info/post/cmake-msvc-and-ninja/). But with some headache.